<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaInserisciRecensioneCliente</class>
 <widget class="QWidget" name="VistaInserisciRecensioneCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inserisci Recensione - CineMax</string>
  </property>
  <widget class="QLabel" name="Sfondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 1,
        x2: 0, y2: 0,
		stop: 0 #211400,
        stop: 1 #7D4B00
    );
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitolo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #D7AA0C;
}</string>
   </property>
   <property name="text">
    <string>Inserisci Recensione</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTipo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C87800;
}</string>
   </property>
   <property name="text">
    <string>Stelle:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelProfilo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Immagini/RecensioneIconCliente.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelTesto">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C87800;
}</string>
   </property>
   <property name="text">
    <string>Testo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConferma">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #963C00;
    color: #FF7800;
    border: 2px solid #502800;
    border-radius: 6px;
}

QPushButton:focus {
    outline: none;
}

QPushButton:hover {
    background-color: #C86400;
}
</string>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAnnulla">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #963C00;
    color: #FF7800;
    border: 2px solid #502800;
    border-radius: 6px;
}

QPushButton:focus {
    outline: none;
}

QPushButton:hover {
    background-color: #C86400;
}
</string>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxStelle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>76</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #321E00;
    color: #965A00;
    border: 1px solid #190E00;
    border-radius: 4px;
}

QComboBox QAbstractItemView {
    background-color: #321E00;
	color: #965A00;
    border: 1px solid #190E00;
	border-radius: 4px;
    selection-background-color: #C86400;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #C86400;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditTesto">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: #321E00;
    color: #965A00;
    border: 1px solid #190E00;
    border-radius: 4px;
}

QTextEdit QScrollBar:vertical {
        background: #B46E00;
        border: 2px solid #190E00;
        border-radius: 4px;
        width: 21px;
        margin: 5px;
}

QTextEdit QScrollBar::handle:vertical {
	background: #965A00;
    min-height: 20px;
    border-radius: 4px;
}

QTextEdit QScrollBar::handle:vertical:hover {
     background: #C86400;
}

QTextEdit QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
	background: none;
    border: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreStelle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFF00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreTesto">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFF00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
